src/ign.cc

```diff
@@ -267,3 +272,87 @@ extern "C" SDFORMAT_VISIBLE int cmdGraph(
 
   return 0;
 }
+
+//////////////////////////////////////////////////
+extern "C" SDFORMAT_VISIBLE int cmdInertialStats(
+    const char *_path)
+{
+  if (!sdf::filesystem::exists(_path))
+  {
+    std::cerr << "Error: File [" << _path << "] does not exist.\n";
+    return -1;
+  }
+
+  sdf::Root root;
+  if (!root.Load(_path).empty()) {
+    return -1;
+  }
+
```

Comment:
It may be good to check `root.WorldCount()` to verify it's a model file and not a world file

Reply:
Done